static/docs/commands-reference/pull.md

```diff
@@ -84,11 +84,11 @@ reflinks or hardlinks to put it in the workspace without copying. See
   behavior from the `dvc checkout` command because `dvc pull` in effect performs
   a _checkout_ after downloading files.
 
-- `-R`, `--recursive` - `targets` values is expected to be a directory path.
-  Determines the files to download by searching the named directory and its
-  subdirectories for DVC-files to download data for. Along with providing a
-  `target`, or `target` along with `--with-deps` it is yet another way to cut
-  the scope of DVC-files to download.
+- `-R`, `--recursive` - `targets` values is expected to be a directory path
```

Comment:
very hard to understand what's going on here. The primary intention of `-R` is to accept directories and search for the DVC-files inside them. It accepts files along with directories for simplicity. For directories it goes down recursively, for files it behaves in a regular way.

Reply:
Is the message now ok @shcheklein?

I will make the subsequent changes tomorrow

Reply:
No, not really :( Can you try to explain me in this thread what this option is about? In a few simple sentences. Like 1, 2, 3 ?

Reply:
For dvc repro -R

1.  Targets can be a file or directory
2.  What it does is reproduce all stages for specified target
3.  --with-deps cuts scopes of DVC-ffiles being reproduced or staged

When I tried locally, there was no difference in `dvc repro` and `dvc repro -R` for a specific target. I don't correctly understand the use case of `-R` flag. Can you please explain @shcheklein